Library/Homebrew/cmd/config.rb

```diff
@@ -131,8 +131,9 @@ def describe_system_ruby
 
   def describe_java
     unless OS.mac?
-      return "N/A" if which("java").nil?
-      java = `java -version 2>&1`.lines.first.chomp
+      return "N/A" if (which "java").nil?
+      java = `java -version 2>&1`
+      java = java.lines.first.chomp
```

Comment:
If it's just stylistic, please remove this chunk. Did you figure out what the root cause of the error was?